.werft/values.k3sWsCluster.yaml

```diff
@@ -0,0 +1,175 @@
+installation:
```

Comment:
This file is copy and modify of `values.dev.yaml` I have scrapped some fields and added some to disable few components.

I have also updated the affinities